pages/solidity-api/ChipRegistry.md

```diff
@@ -178,7 +179,7 @@ function updateProjectMerkleRoot(contract IProjectRegistrar _projectRegistrar, b
 ```
 
 _Update the merkle root of a project enrollment. This function is only callable by the project's public key. This function
-will revert if the project has already claimed a chip from this enrollment or the 7-day update time period has elapsed. New URI
+will revert if the project has already claimed a chip from this enrollment or the 7-day update time period (find link to this) has elapsed. New URI
 is required because IPFS records are immutable so changing the merkle root would require a new IPFS record._
```

Comment:
where is this 7-day update time period declared? is this customizable? why 7-days?